src/sagas/wallet.js

```diff
@@ -202,6 +202,13 @@ export function* startWallet(action) {
 
       // Yield the same action so it will now load on the old facade
       yield put(action);
+    } else {
+      // NOTE: perhaps we should provide a reset option to the user.
+      // A reset option would be nice to have because the user may
+      // have configured a custom network.
```

Comment:
What would happen if this is the case? He would have to uninstall the app?

Maybe we should create this KTLO and prioritize it

Reply:
Ok! I will work a KTLO, then.

Reply:
I registered the KTLO and here is the issue: https://github.com/HathorNetwork/hathor-wallet-mobile/issues/368

Reply:
I agree that this is an important KTLO, and we shouldn't consider the project finished without it. I used to have this problem a lot with the desktop wallet when I was testing with a `localhost` network, and then I wanted to use the wallet later and my localhost was not up anymore.

Reply:
Implemented at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/365/commits/64c7f14a3dc35bc11309638ed254de7658e2922d